Mod0/syllabus.md

```diff
@@ -77,23 +80,16 @@ There will be no “gotchas”. The rubric that follows communicates what we car
 ## Required Supplies
 This may be your first time learning and/or working in a remote setting. There are a few key tools we recommend to set yourself up for success as a remote student. 
 
-- **External Monitor/Display:** There are plenty of great used options available on eBay, Facebook Marketplace, Craigslist, etc. All you really need is something that fits in your workspace and can connect to your MacBook! If you are unable to acquire an external monitor prior to Mod 1, please DM the Mod 0 Lead on Slack and we can discuss other options.
-- **USB-C to HDMI Cable:** This cable will allow you to connect your MacBook to your external monitor/display. If you are purchasing a new monitor, it will likely come with the cable to you, but you may need to aquire one.
+- **External Monitor/Display:** You will receive an external monitor along with your laptop. You will appreciate the additional screen space using all the tools as you navigate your new career. An HDMI-to-HDMI cable (i.e. full-size HDMI plug on both ends) is included in the monitor box to connect your laptop to the monitor.
 - **Dedicated Workspace:** It is crucial to have a workspace that has access to a reliable internet connection and is both quiet and free from distractions. Take time before Mod 0 to build your workspace and experiment with different setups until you find a configuration that works for you.
-- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops.
+- **Notebook + Writing Utensils:** While you will be working on your laptop while you code, work on projects, etc., we recommend using a physical notebook to take notes during class and workshops. There are apps available 
```

Comment:
I think this sentence needs to be completed :)

Reply:
Nope, that was planned ;-) Kidding, I'm on it.